src/main/java/uk/gov/companieshouse/officer/delta/processor/tranformer/TransformerUtils.java

```diff
@@ -100,7 +100,12 @@ public static String encode(String plain) throws NonRetryableErrorException {
         return base64Encode(sha1Digest(plain)).replace("=", "");
     }
 
-    public static String lookupOfficeRole(String kind) {
+    public static String lookupOfficeRole(String kind, String corpInd) {
```

Comment:
Just noticed there's a typo in this method name

Reply:
Will fix this 👍